core-chart/values.yaml

```diff
@@ -3,6 +3,10 @@
 # Declare variables to be passed into your templates.
 
 
+# Name of the ITS control plane to use for transport-controller (optional)
+# If not set, auto-discovery will be used (legacy behavior)
+ITSName: ""
```

Comment:
This should not be a global variable. It is already designed to be specified per WDS.

Reply:
Look at my recent comments on #3192, I think that I found the real bug.